met-web/src/components/comments/admin/review/CommentReview.tsx

```diff
@@ -168,6 +187,25 @@ const CommentReview = () => {
     const defaultVerdict = comment_status_id !== CommentStatus.Pending ? comment_status_id : CommentStatus.Approved;
     return (
         <MetPageGridContainer>
+            <EmailPreviewModal
+                open={open}
+                handleClose={() => setOpen(false)}
+                header={'Your comment on (Engagement name) needs to be edited'}
+                emailText={[
```

Comment:
could we discuss this pattern in a meeting. I suggest we emulate Material UIs pattern for customizable rendering like renderInput and renderOption below in AutoComplete
![image](https://user-images.githubusercontent.com/91914654/215151621-438fce7f-7fe8-4556-80c3-75fefff584df.png)